RichardSzalay.MockHttp.Shared/MockHttpMessageHandler.cs

```diff
@@ -86,10 +86,17 @@ public void Flush(int count)
         /// <summary>
         /// Creates an HttpClient instance using this MockHttpMessageHandler
         /// </summary>
+        /// <param name="initalize">(Optional) Allows the option to confiure the Http client before creation</param>
         /// <returns>An instance of HttpClient that can be used to send HTTP request against the configuration of this mock handler</returns>
-        public HttpClient ToHttpClient()
+        public HttpClient ToHttpClient(Action<HttpClient> initalize = null)
```

Comment:
This is a binary-level breaking change because it adds a parameter. (Even though it provides a default value.) [Reference](https://stackoverflow.com/a/23517490/6827589)

You could provide the new overload in addition to the existing `ToHttpClient` overload to avoid a breaking change:
```cs
public HttpClient ToHttpClient() { ... }

public HttpClient ToHttpClient(Action<HttpClient> initalize) { ... }
```